### PR TITLE
[TIMOB-25860] Android: Update support libraries to 27.1.1

### PR DESCRIPTION
**Pull request by garymathews**
- Update Android Support Libraries to `27.1.1`
- Include `exifinterface` support library

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-25860)

### CODE REVIEW

**Comment by jquick-axway** [comment]
This PR also covers this ticket:
https://jira.appcelerator.org/browse/TIMOB-25852

**Comment by garymathews** [comment]
##### NOTE
We should update to `27.1.1`

**Comment by jquick-axway** [comment]
Can you also do the following please?
- Update build tools version to 27 (it's currently 26)
- Add Google's EXIF support library (needed by future refactoring of image loading code)

**Comment by lokeshchdhry** [comment]
@garymathews , Are we planning to do the above changes to this PR ?

**Comment by jquick-axway** [comment]
Thanks for adding the EXIF library!

File: android/package.json [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
 	},
 	"minSDKVersion": "16",
-	"compileSDKVersion": "26",
+	"compileSDKVersion": "27",
 	"vendorDependencies": {
-		"android sdk": ">=23.x <=25.x",
-		"android build tools": "26.x",
-		"android platform tools": "26.x",
+		"android sdk": ">=23.x <=27.x",
+		"android build tools": "<=26.x",
```
Comment (jquick-axway) [inline android/package.json]: Can you update build-tools version to "27.x" please?
Or perhaps it should be ">=25.x <=27.x".
You said v25 is the max version IntelliJ supports, right?

**Review by jquick-axway — APPROVED** [review]
CR: Pass

**Comment by lokeshchdhry** [comment]
FR Passed.

Verified android manifest includes : 
``<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="27"/>``

Ran Kitchensink V2 & no issues seen. 

Studio Ver:     5.1.0.201804230827
SDK Ver:        7.2.0 local build
OS Ver:         10.13.4
Xcode Ver:      Xcode 9.3
Appc NPM:       4.2.13
Appc CLI:       7.0.3
Daemon Ver:     1.1.1
Ti CLI Ver:     5.1.0
Alloy Ver:      1.12.0
Node Ver:       8.9.1
NPM Ver:        5.5.1
Java Ver:       10
Devices:        ⇨ google Nexus 6P --- Android 8.1.0
                ⇨ google Nexus 5 --- Android 6.0.1
Emulator:       ⇨ Android 4.1.2

**Pull request merged**